src/frameworks/Carbon/HIToolbox/include/HIToolbox/IBCarbonRuntime.h

```diff
@@ -0,0 +1,37 @@
+#ifndef _Carbon_IB_Carbon_Runtime_H_
+#define _Carbon_IB_Carbon_Runtime_H_
+
+#include <CoreFoundation/CoreFoundation.h>
+#include <CarbonCore/MacErrors.h>
+#include <CoreFoundation/CFBase.h>
+#include <HIToolbox/Menus.h>
+#include "../../src/MacWindows.h" // TODO: put WindowRef where it actually goes
```

Comment:
I'm not sure why `MacWindows.h` is not included in the header folder...

Can you create a separate commit for moving `MacWindows.h` to the header folder (and fixing the includes that reference `MacWindows.h`)?

Reply:
Two problems:

- WindowRef is supposed to be in AppKit, and I'm not sure what it's doing here
- `<HIToolbox/MacWindows.h>` isn't in any list of header files I've found (I was thinking I could rename it to `<HIToolbox/_MacWindows.h>`, so it's really not much of a problem

Reply:
found documentation for MacWindows.h. https://web.archive.org/web/20160206202121/http://mirror.informatimago.com/next/developer.apple.com/documentation/Carbon/reference/Window_Manager/wind_mgr_ref/function_group_21.html

Reply:
I was going to just add all stubs to that header since I was there anyway, but as it turns out in Leopard, theres more than 500 symbols, so I'll not do that in this PR.

Reply:
> 500 symbols, so I'll not do that in this PR

Honestly, it would be a nightmare for me to verify all 500 symbols at once... I rather do it when we find an app that needs them.